README.md

```diff
@@ -3,7 +3,7 @@
 The XRP Ledger is a decentralized cryptographic ledger powered by a network of peer-to-peer servers. The XRP Ledger uses a novel Byzantine Fault Tolerant consensus algorithm to settle and record transactions in a secure distributed database without a central operator.
 
 ## XRP
-XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide. XRP is traded on the open-market and is available for anyone to access. The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem.  Ripple uses XRP the help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today.
+XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide. XRP is traded on the open-market and is available for anyone to access. The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem.  Ripple uses XRP to help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today.
```

Comment:
Is "counterparty-free" better? Or perhaps something more extensive:

>XRP is the public digital asset, native to the XRP Ledger. Designed as a fast and efficient bridge for the many different currencies in use worldwide, XRP is free of counterparty risk and available for anyone to use. The XRP Ledger ws created [...]

Reply:
Would be fine with "counterparty-free" here. I think I prefer the original wording to Nik's longer rewrite (sorry, @nbougalis!) though.

Reply:
No need to apologize. I've been accused of being many things; laconic isn't one of them.